sql/2025/performance/lcp_resource_type.sql

```diff
@@ -0,0 +1,33 @@
+# We are unable to track LCP for video elements: https://issues.chromium.org/issues/364860066
+# TODO: It looks like this ^^ was resolved 2024-11-27, so does that mean we can track LCP for video now?
```

Comment:
❓ Is this comment about being unable to track LCP for video elements still relevant?

If we are tracking LCP for video elements now, does that mean I should change the `CASE` below to also account for videos?